java/core/src/main/java/co/worklytics/psoxy/di/PsoxyModule.java

```diff
@@ -0,0 +1,40 @@
+package co.worklytics.psoxy.di;
```

Comment:
I find naming package `di` like this is rather weird ... why not just put it at top-level?

Reply:
I prefer not to mix it with the "real" code. DI is just glue for the app, you can really avoid the package while scanning contents, so I prefer to put it in their on package. But whatever.